src/main/kotlin/io/openfuture/api/config/propety/StateProperties.kt

```diff
@@ -0,0 +1,12 @@
+package io.openfuture.api.config.propety
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.stereotype.Component
+import org.springframework.validation.annotation.Validated
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.NotNull
+
+@ConfigurationProperties(prefix = "state")
+@Validated
```

Comment:
Value marked as not null and not blank but accept null strings

Reply:
Yes, its ok, we should expect null but validate it and give clear error message. Also we already did in Open State you can check